runtime/cudaq/matrix_op.h

```diff
@@ -176,12 +228,42 @@ class matrix_handler : public operator_handler {
 
   // predefined operators
 
+  /// @brief Constructs a product operator representing a number operator for
+  /// the given degree.
+  /// @param degree : The degree or power to which the number operator is
+  /// constructed.
+  /// @return A matrix_handler instance that encapsulates the
+  /// constructed number operator matrix.
   static matrix_handler number(std::size_t degree);
+  /// @brief Creates a parity operator using a product operator with
+  /// matrix_handler.
+  /// @param degree : The degree of the parity transformation, which may
+  /// correspond to the number of qubits or the order of the operation.
```

Comment:
Number of qubits here is also incorrect. It acts on a single degree of freedom (which is not necessarily a qubit or a particle for that matter)."degree: the degree of freedom that the parity operator acts on" is a correct and sufficient parameter description.